### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by shadeov**
I completely redid the `README.md`. Now it's much easier to read and understand.

### CODE REVIEW

**Comment by cocool97** [comment]
That's a huge improvement ! 

Just a thing, project image is hosted at <https://github.com/cocool97/TorrServer/assets/144587546/c32b1f99-f6b1-47e3-84ab-c01e954e4706>, could you please change it to get it from main project ? This would need modifications before merging in `master` project branch.

After that I'll merge it ;)

File: README.md [hunk 1]
```diff
@@ -1,74 +1,182 @@
...
+- Install [Golang](https://golang.org/doc/install) 1.16+
+- Goto dir to source
+- Run build script under linux `build-all.sh`
+
+#### Web
+
+- Install **npm** and **yarn**
+- ```bash
+  yarn build
+  ```
```
Comment (cocool97) [inline README.md:173]: Maybe here, instead of having a code block, 
```suggestion
- Run `yarn build`
```

File: README.md [hunk 2]
```diff
@@ -80,19 +188,23 @@ cd server; swag init -g web/server.go
...
-#### API Docs
+### API Docs
 
 API documentation is hosted as Swagger format available at path `/swagger/index.html`.
 
-#### API Authentication
+### API Authentication
 
-The user data file should be located near to the settings. Basic auth, read more in wiki <https://en.wikipedia.org/wiki/Basic_access_authentication>.
+The user data file should be located near to the settings. Basic auth, read more in wiki https://en.wikipedia.org/wiki/Basic_access_authentication.
```
Comment (cocool97) [inline README.md:205]: Shouldn't link be with '<' '>' in Markdown ?

**Review by cocool97 — CHANGES_REQUESTED** [review]

**Comment by shadeov** [comment]
> That's a huge improvement !
> 
> Just a thing, project image is hosted at https://github.com/cocool97/TorrServer/assets/144587546/c32b1f99-f6b1-47e3-84ab-c01e954e4706, could you please change it to get it from main project ? This would need modifications before merging in `master` project branch.
> 
> After that I'll merge it ;)

This link is automatically generated when inserting an image in the ReadMe editing mode within the GitHub editor and is not actually related to the repository. But since the question arose, I did it.

**Comment by YouROK** [comment]
Поправь пожалуйста раздел с установкой, альтернативная установка. Она как будто к маку относится

**Pull request merged**